### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by aviatesk**
tries to do basic updates:
- [x] move to Project.toml and add a description for how to build docs locally
- [x] update how/where to report issues and communicate with us
- [x] remove no-longer used appveyor

I have a bunch of improvement ideas of our docs other than this, but let me do them in another PR. (as discussed in https://github.com/JunoLab/Juno.jl/issues/419)

### CODE REVIEW

File: docs/src/devdocs/contributor_guide.md [hunk 1]
```diff
@@ -4,26 +4,24 @@
...
+open an [issue on Juno.jl GitHub repository](https://github.com/JunoLab/Juno.jl/issues)
+with an appropriate tag -- we're always happy to help people flesh out their ideas or get unstuck on problems.
+Another good place to communicate with Juno developers is [our Slack channel](https://julialang.slack.com/archives/C7JT7HQAD).
 Come by and we can chat about your ideas.
 
 If you look over the GitHub issues for the various packages, you may notice some labelled
-[up for
-grabs](https://github.com/JunoLab/atom-julia-client/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22).
-These are features or bugs for which the implementation or fix is reasonably straightforward –
+[up for grabs](https://github.com/JunoLab/atom-julia-client/issues?q=is%3Aopen+is%3Aissue+label%3A%22up+for+grabs%22).
```
Comment (pfitzseb) [inline docs/src/devdocs/contributor_guide.md]:
```suggestion
[up for grabs](https://github.com/search?q=org%3AJunoLab+is%3Aissue+label%3A%22up+for+grabs%22+is%3Aopen).
```

File: docs/src/index.md [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
+powerful tools to help you develop code. Juno is built on [Atom](https://atom.io), a text
+editor provided by GitHub. Juno consists of both Julia and Atom packages in order to
 add Julia-specific enhancements, such as syntax highlighting, a plot pane,
 integration with Julia's debugger, a console for running code, and much more.
 
 ### Installation
 
 To install Juno, please see the [Installation Instructions](@ref). These installation
-instructions should work for Windows, Mac OSX, and Linux on the current Julia release (v0.6).
+instructions should work for Windows, Mac OSX, and Linux on the current Julia release (v1.2).
```
Comment (pfitzseb) [inline docs/src/index.md]:
```suggestion
instructions should work for Windows, Mac OSX, and Linux on the current Julia release (v1).
```
so people don't think we're incompatible with e.g. v1.0.

File: docs/src/man/faq.md [hunk 3]
```diff
@@ -2,8 +2,13 @@
 
 ### I am having issues, where do I leave a bug report?
 
-Please leave bug reports either [at the discussion board](https://discourse.julialang.org)
-or [at the Juno.jl Github repository](https://github.com/JunoLab/Juno.jl/issues).
+Please leave bug reports either at
+[Julia's discussion forum under the `Tooling ▶ Juno` category](https://discourse.julialang.org/c/tools/juno/l/latest)
+or [the Juno.jl GitHub repository](https://github.com/JunoLab/Juno.jl/issues).
```
Comment (pfitzseb) [inline docs/src/man/faq.md]: Mind switching the order of those? I'd rather have bug reports on GitHub than Discourse, I think.

**Review by pfitzseb — APPROVED** [review]

File: docs/src/man/faq.md (hunk 3 above)
Comment (aviatesk) [reply docs/src/man/faq.md]: Yeah I prefer GitHub for having bug reports too. Do you want even not to mention Discourse here ?

Comment (aviatesk) [reply docs/src/man/faq.md]: Actually I want to have issue reports only on Juno.jl repo or slack for lightweight questions. But I know Discourse is a good to place to announce and some people want to report issues on there.
Just let me know your preference, @pfitzseb :)

Comment (pfitzseb) [reply docs/src/man/faq.md]: Just switch them around, I think. Discourse will be used no matter what we say here ;)

Comment (aviatesk) [reply docs/src/man/faq.md]: thx for your response, okay I will modify around this and going to merge this then 😁

**Comment by aviatesk** [comment]
@pfitzseb Are b136f4c and c11244a okay for you ?

**Pull request merged**